src/yz_wm_index.erl

```diff
@@ -179,16 +182,26 @@ delete_resource(RD, S) ->
 
 %% Responds to a PUT request by creating an index and setting the
 %% index flag for the "index" name given in the route. Returns 204 if
-%% created, or 409 if an index of that name already exists. Returns a
-%% 400 error if the schema does not exist.
+%% created. Returns a 400 error if the schema does not exist or the index name
+%% contains an invalid char or a bad n_val is given or if a request doesn't
+%% complete within a set timeout.
```

Comment:
pendantic grammar police: use commas to replace some "or"s. e.g. "Returns a 400 error if the schema does not exist, the index name contains an invalid char, a bad n_val is give, or if a request doesn't complete within a set timeout."